### DISCUSSION TITLE
How to use this WindiCSS with Quasar Framework?

### DISCUSSION THREAD

**Discussion by rashidpathiyil** [windicss/windicss#482] [Q&A] [answered]
A big WindiCSS fan would like to use WindiCSS  with [Quasar Framework](https://quasar.dev/)
What is the best way to config WindiCSS with Quasar?

**Comment by cyfung1031** [comment]
any prefix option like tailwindcss?

**Reply by Rolanddoda** [reply]
Any update on this ?

**Comment by antfu** [comment, marked as answer]
https://github.com/windicss/windicss-webpack-plugin
https://quasar.dev/quasar-cli/handling-webpack#usage-with-quasar-conf-js

I guess you could install the webpack-plugin in the Quasar config
